### ISSUE TITLE
Converting Caffe to PyTorch

### ISSUE THREAD

**Issue by ilichev-andrey** [pytorch/pytorch#3850]
Hello. I want to convert the model from Caffe to PyTorch. Model is resnet50_1by2 custom dataset.

**Variant 1**. Using torch and loadcaffe converted model.
**Result**: not all layers were converted.
**Example**: 
> -- warning: module 'bn_stage0_block0_branch2c [type BatchNorm]' not found
> -- warning: module 'scale_stage0_block0_branch2c [type Scale]' not found
> -- warning: module 'eltwise_stage0_block0 [type Eltwise]' not found

**Variant 2**. Install Caffe and convert params to numpy. Load numpy arrays to PyTorch.
**Example**:
Caffe -> PyTorch
> conv_1_0 -> conv_1.weight
> conv_1_1 -> conv_1.bias
> bn_1_0 -> bn_1.running_mean
> bn_1_1 -> bn_1.running_var
> scale_1_0 -> bn_1.weight
> scale_1_1 -> bn_1.bias

**Result**: All the sizes of layers coincide, architecture too. All weights correspond! PyTorch model working! But when I make the profiling, the result is different after each layer and as a result the values are completely wrong.

What could be wrong with the **Variant 2**?

P.S. Pictures for testing are the same, normalization too. 3x224x224 (0, 255). 
Caffe input BGR image, and PyTorch input BGR image.

Full profiling:

1) Export caffe params:
briefly: [caffe_params](https://drive.google.com/open?id=1O1Hdy0FYe0DE2taW2PWltmGun4XXYGxN) (split(' '))
full: [caffe_params_ext](https://drive.google.com/open?id=1AYtmULE3mxP4FpujzRv7KgmtehJ_hzL5)

2) Merge caffe and pytorch layers:
[merge](https://drive.google.com/open?id=1IRDrSml1urz2SH0uwoi2IYlECfq-yElY) (split(' '))

3) Initialization parameters in pytorch and export params to numpy:
briefly: [pytorch_params](https://drive.google.com/open?id=1rysuSQUSt2ckWfezUgQQR3GptXFzZhCK) (split(' '))
full: [pytorch_params_ext](https://drive.google.com/open?id=1aChkwGOmVs3WaI8Xu1XZOkketsR3XcaA)

4) Testing model:
The initial and final layers recorded
[caffe_outputs](https://drive.google.com/open?id=12HzBQO9r8qJRmZ7TjU5Msf-a8Ui3qxCL) 
[pytorch_outputs](https://drive.google.com/open?id=1sBYSRJqjD38fYRsjJ6YO5bv351uyJ_rb)

**Comment by apaszke** [comment]
Note that we're using GitHub issues for bug reports only, and all questions should be posted on [our forums](discuss.pytorch.org).
